src/app.js

```diff
@@ -1 +1,40 @@
+/* eslint-disable no-console */
 'use strict';
+
+const readline = require('readline');
+const fs = require('fs');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const questionForUser = (question) => {
+  return new Promise((resolve) => {
+    terminal.question(question, (answer) => {
+      if (!answer.length) {
+        console.log('Enter at least one character');
+
+        return questionForUser(question);
+      }
+      resolve(answer);
+    });
+  });
+};
```

Comment:
Let's use `'readline/promises'` library instead. It's available from 17+ node version